src/addon-manager-proxy.ts

```diff
@@ -161,23 +226,26 @@ class AddonManagerProxy extends EventEmitter {
    * @method onMsg
    * Called whenever a message is received from the gateway.
    */
-  onMsg(msg) {
-    this.verbose && console.log('AddonManagerProxy: Rcvd:', msg);
+  onMsg(genericMsg: Message2): void {
+    this.verbose && console.log('AddonManagerProxy: Rcvd:', genericMsg);
 
-    switch (msg.messageType) {
+    switch (genericMsg.messageType) {
       case MessageType.PLUGIN_UNLOAD_REQUEST:
         this.unloadPlugin();
         return;
 
       case MessageType.API_HANDLER_UNLOAD_REQUEST: {
+        // APIHandlerUnloadRequest was somehow not generated
```

Comment:
Can we fix this? Seems weird.

Reply:
Fix: https://github.com/WebThingsIO/gateway-addon-ipc-schema/pull/8

Reply:
Done